Replanter/Replanter.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <AssemblyName>Replanter</AssemblyName>
     <RootNamespace>Replanter</RootNamespace>
-    <Version>1.1.1</Version>
+    <Version>1.1.3-unofficial.1-jacquesdereims</Version>
```

Comment:
(This shouldn't be in the pull request, since it would add the unofficial version number to the official repo.)

Reply:
Reverted version number to take out unofficial version, sorry!